File: 1252-break-a-palindrome/break-a-palindrome.py
class Solution:
    def breakPalindrome(self, palindrome: str) -> str:
        if len(palindrome) == 1: return ""
        ans = ""
        for i in range(len(palindrome)):
            if palindrome[i] != 'a':
                ans+='a'
                ans+=palindrome[i+1:]
                if ans != ans[::-1]:
                    return ans
                else:
                    break
            ans+=palindrome[i]
        return palindrome[:-1]+'b'

File: 1252-break-a-palindrome/break-a-palindrome.js
/**
 * @param {string} palindrome
 * @return {string}
 */
var breakPalindrome = function(palindrome) {
    if (palindrome.length < 2) return ""
    let str = ""
    for (let i = 0; i < palindrome.length; i++) {
        if (palindrome[i] != 'a') {
            str += 'a'
            str += palindrome.slice(0, i) + palindrome.slice(i + 1)
            let reverseStr = str.split('').reverse().join('')
            if (str !== reverseStr) {
                return str
            }
        }
    }
    return palindrome.slice(0, palindrome.length - 1) + 'b'
};